content/en/blog/_posts/2021-00-00-kpng-specialized-proxiers.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: blog
+title: "Use KPNG to write specialized kube-proxiers"
+date: 2021-09-00
+slug: use-kpng-to-write-specialized-kube-proxiers
+---
+
+Authors: Lars Ekman, Ericsson
+
+The post describes how you can create a specialized proxier using
```

Comment:
```suggestion
The post shows you how anyone can create a specialized service kube-proxy style network proxier, using
```

Reply:
I leave out the "easily". Granted that the API stuff hadled by kpng is easy, but load-balancing is certainly not :smile: